lib/ClangImporter/SortedCFDatabase.def.gyb

```diff
@@ -27,7 +27,7 @@ lineForName = {}
 
 # Load the data file.
 with open(CFDatabaseFile, 'rb') as f:
-  for line in f:
+  for line in [line_bytes.decode('utf-8') for line_bytes in f]:
```

Comment:
That was a good suggestion. I've never written Python before and I hadn't noticed the difference between the parenthesis and square brackets.

Though in the end I actually found there is a `codecs` module that does what I am looking for explicitly and I've refactored the code to make use of that instead.